server/Containerfile

```diff
@@ -1,7 +1,7 @@
 # Test container image that represents a remote managed server which connects
 # to consoledot. This illustrates the minimum dependencies.
-FROM debian:bookworm
 
-RUN apt-get update && \
-    apt-get install -y cockpit-bridge cockpit-system ca-certificates python3 && \
-    apt-get clean
+ARG tag=stream9
```

Comment:
Nice! Cool that we test 8/9 now!

Reply:
But this is a bit inconsistent? By default we test on centos8 in the Makefile so wouldn't that make a better default?

Reply:
We don't, it's centos9 -- that's precisely what this ARG says. But indeed it's confusing to have webconsoleserver:latest be centos9, I'll make the tags consistent.

Reply:
Done